### ISSUE TITLE
expanding image instead of inpainting

### ISSUE THREAD

**Issue by jaroslaw-weber** [JiahuiYu/generative_inpainting#205]
I checked out your demo and it works nicely for small distortions. Did you try use this model for "outpainting"? I mean extending the field of view by providing photo with frame and mask the frame. Something like "extend" in snapseed. I tested it on your demo but the results were not so good. Do you think finetuning the model is enough or does the model architecture need to be changed?

**Comment by JiahuiYu** [comment]
@jaroslaw-weber I would expect good results for "outpainting". For that, I would suggest to train from scratch. You will need to change the mask generation related code. I would also expect current hyper-parameters good enough.
